crypto/crypto.go

```diff
@@ -43,12 +43,9 @@ import (
 	"golang.org/x/crypto/ripemd160"
 )
 
-var secp256k1n *big.Int
-
 func init() {
 	// specify the params for the s256 curve
-	ecies.AddParamsForCurve(S256(), ecies.ECIES_AES128_SHA256)
-	secp256k1n = common.String2Big("0xfffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364141")
+	ecies.AddParamsForCurve(secp256k1.S256(), ecies.ECIES_AES128_SHA256)
```

Comment:
I think this is no longer required since you added an entry for secp256k1 in the default params registry.